scripts/devnet/forest_ci.dockerfile

```diff
@@ -1,4 +1,5 @@
-FROM ubuntu:22.04
+# The version has to match Github CI runner images
```

Comment:
We'd better use explict ubuntu version in [CI].yml otherwise CI might be broken someday when `ubuntu-latest` starts pointing to `ubuntu-24.04`